tensorflow/core/kernels/training_ops.cc

```diff
@@ -24,8 +24,8 @@ limitations under the License.
 #include "tensorflow/core/framework/bounds_check.h"
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/register_types.h"
-#include "tensorflow/core/kernels/training_op_helpers.h"
```

Comment:
`tensorflow/core/framework/training_op_helpers.cc:18:10: error: module //
/tensorflow/core:training_op_helpers does not depend on a module exporting '
/tensorflow/core/util/ptr_util.h`